projects/driver_controls/src/power_fsm.c

```diff
@@ -7,11 +7,22 @@
 // Power FSM state definitions
 
 FSM_DECLARE_STATE(state_off);
+FSM_DECLARE_STATE(state_off_brake);
+FSM_DECLARE_STATE(state_charging);
 FSM_DECLARE_STATE(state_on);
 
 // Power FSM transition table definitions
 
 FSM_STATE_TRANSITION(state_off) {
+  FSM_ADD_TRANSITION(INPUT_EVENT_POWER, state_charging);
+  FSM_ADD_TRANSITION(INPUT_EVENT_MECHANICAL_BRAKE_PRESSED, state_off_brake);
+}
+
+FSM_STATE_TRANSITION(state_off_brake) {
+  FSM_ADD_TRANSITION(INPUT_EVENT_POWER, state_on);
```

Comment:
Shouldn't there be a state transition back to `state_off` on mechanical brake released?

Reply:
@Derpalooza

Reply:
That was a mistake. I can get that fixed.